docfx_project/ziti/clients/linux.md

```diff
@@ -64,6 +64,16 @@ If any interfaces have a wildcard routing domain configured, `ziti-edge-tunnel`
 --dns-ip-range <ip range>
 ```
 
+#### How does `ziti-edge-tunnel run` configure nameservers?
+
+`ziti-edge-tunnel run` provides a built-in nameserver that will answer queries that exactly match authorized OpenZiti services' intercept domain names, and will respond with a hard-fail `NXDOMAIN` code if the query does not match an authorized service.
+
+Optionally, you may enable DNS recursion by specifying an upstream nameserver to answer queries for other domain names that are not services' intercept domain names: `ziti-edge-tunnel run --dns-upstream 208.67.222.222`.
+
+`ziti-edge-tunnel` uses the `libsystemd` D-Bus RPC client and will try to configure the OS's resolvers with `systemd-resolved`. If that's not possible for any reason then `ziti-edge-tunnel run` will fall back to shell commands like `resolvectl`. If those too do not succeed then `ziti-edge-tunnel run` will attempt to modify `/etc/resolv.conf` directly to install the built-in nameserver as the primary resolver.
+
+`process_host_req()` in [`/lib/ziti-tunnel-cbs/ziti_dns.c`](/lib/ziti-tunnel-cbs/ziti_dns.c) looks up the queried hostname in the internal map. If the entry exists it returns the answer and sets query status to `NO_ERROR`. If it does not exist in the map, it sends the query to an upstream DNS server if available, and otherwise sets the query status to `REFUSE`. This implies to the caller they *should* keep trying to resolve the domain name with other nameservers.

```

Comment:
Leave off the implementation details like what file and what function or that it uses an internal map. Make this more generic and simpler. Personally, I'd remove this whole paragraph. It's not adding value to me. maybe it just needs to be rewritten? 

I'm leaning on removing this entirely